eng/pipelines/libraries/build-job.yml

```diff
@@ -51,6 +52,11 @@ jobs:
       name: build
       displayName: 'Build'
 
+      ${{ if and(ne(parameters.liveRuntimeBuildConfig, ''), eq(parameters.runTests, true)) }}:
```

Comment:
`ne(parameters.liveRuntimeBuildConfig, '')`, when would that ever be false?

Reply:
allConfigurations build and netfx build

Reply:
why the allconfigurations build?

Reply:
because the allConfigurations build doesn't need coreclr. It runs the package tests against a previous built Microsoft.NETCore.App at the moment and because we just need corelib to build the projects which are built via the P2P